egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+set -e
+set -o pipefail
+
+# This is fisher chain recipe for training a model on a subset of around
+# 100-300 hours of supervised data.
+# This system uses phone LM to model UNK.
+# local/semisup/run_50k.sh and local/semisup/run_100k.sh show how to call this.
+
+# train_set                 train_sup15k           train_sup50k           train_sup
+# ivector_train_set         semisup15k_100k_250k   semisup50k_100k_250k   train_sup
+# WER on dev                27.75                  21.41                  19.23
+# WER on test               27.24                  21.03                  19.01
+# Final train prob          -0.0959                -0.1035                -0.1224
+# Final valid prob          -0.1823                -0.1667                -0.1503
+# Final train prob (xent)   -1.9246                -1.5926                -1.6454
+# Final valid prob (xent)   -2.1873                -1.7990                -1.7107
+
+# train_set                           semisup15k_100k_250k    semisup50k_100k_250k    semisup100k_250k
```

Comment:
are all of these results still part of the scripts?  remove any that are not.
If these lines came from compare_wer.sh it would be nice if you could show the corresponding command line.
and an explanation of the naming convention would be nice too.... in semisupX_Y_Z, it's not clear what X, Y and Z are.